pkg/meta/model/job_args.go

```diff
@@ -1343,137 +1351,48 @@ type IndexArg struct {
 	IsGlobal bool  `json:"is_global,omitempty"`
 }
 
-// DropIndexArgs is the argument for drop index.
-// IndexIDs may have different length with IndexNames and IfExists.
-// When it's used in dropping index, length of IndexNames, IfExists and IndexIDs are 1.
-// When it's used in rolling back add indexes,
-// len(IndexNames) and len(IfExists) equals to the number of merged added index jobs,
-// while IndexIDs stores the partition IDs of the corresponding table.
-type DropIndexArgs struct {
-	// Only IndexName, IfExists and IsVector fields are used currently.
-	// IsVector is only used in drop index jobs, not rollback add index jobs.
+// ModifyIndexArgs is the argument for add/drop index jobs.
+type ModifyIndexArgs struct {
 	IndexArgs []*IndexArg `json:"index_args,omitempty"`
 
 	// Belows are used for finished args.
+	// For dropping index, len(IndexIDs) is 1.
+	// While during rolling back add indexes, IndexIDs stores the partition IDs of the corresponding table.
```

Comment:
too bad 🙃

Reply:
> too bad 🙃

You mean the comment, or this field? 🤔

Reply:
Maybe I need to rename these two fields. They are confusing indeed.

Reply:
> IndexIDs stores the partition IDs of the corresponding table.

I mean this.

> Maybe I need to rename these two fields. They are confusing indeed.

if the change is large, you can write a TODO and do it in other pr, this one too large